src/Gui/PreferencePages/DlgSettingsNotificationArea.cpp

```diff
@@ -55,8 +56,17 @@ DlgSettingsNotificationArea::~DlgSettingsNotificationArea() = default;
 
 void DlgSettingsNotificationArea::saveSettings()
 {
-    ui->NotificationAreaEnabled->onSave();
-    ui->NonIntrusiveNotificationsEnabled->onSave();
+    // must be done as very first because we create a new instance of NavigatorStyle
+    // where we set some attributes afterwards
+    ParameterGrp::handle hGrp = App::GetApplication().GetParameterGroupByPath
+        ("User parameter:BaseApp/Preferences/NotificationArea");
+
+    bool isNotificationAreaEnabled = ui->NotificationAreaEnabled->isChecked();
+    hGrp->SetBool("NotificationAreaEnabled", isNotificationAreaEnabled);
+    
+    bool isNonIntrusiveNotificationsEnabled = ui->NotificationAreaEnabled->isChecked();
```

Comment:
Fix trailing whitespace on line 66

Reply:
Fixed, thanks. Feel free to use the "Suggestion" button in GitHub in the future. It's quicker for the reviewer to do it than commenting, and the author can just apply it at the click of a button. Thank you.